service/src/test/java/uk/gov/hmcts/reform/fpl/service/ccd/CoreCaseDataServiceTest.java

```diff
@@ -56,6 +72,30 @@ void shouldMakeAppropriateApiCalls() {
             CASE_TYPE, Long.toString(caseId), true, buildCaseDataContent(eventId, eventToken));
     }
 
+    @Test
+    void shouldReturnMatchingCaseDetailsIdWhenSearchedByExistingCaseId() throws IOException {
+        CaseDetails expectedCaseDetails = populatedCaseDetails();
+
+        when(requestData.authorisation()).thenReturn(userAuthToken);
+
+        when(coreCaseDataApi.getCase(userAuthToken, serviceAuthToken, Long.toString(caseId)))
+            .thenReturn(expectedCaseDetails);
+
+        CaseDetails returnedCaseDetails = service.findCaseDetailsById(Long.toString(caseId));
+
+        assertThat(expectedCaseDetails.getId())
+            .isEqualTo(returnedCaseDetails.getId());
```

Comment:
why relax this assertion?
just do assertThat(expectedCaseDetails).isEqualTo(returnedCaseDetails);
and remove Id from test method name